### PR TITLE
init: add pom and base classes

### PR DESCRIPTION
**Pull request by g4s8**
Add `pom.xml` for project config. Add base classes
for handling smart-http protocol: `GitSlice` is an entrypoint,
other slices will be responsible for receive and upload pack commands.

### CODE REVIEW

File: src/main/java/com/artipie/git/ReceivePackSlice.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ReceivePackSlice() {
+        super(new SliceSimple(new RsWithStatus(RsStatus.NOT_IMPLEMENTED)));
+    }
+
+    /**
+     * A slice to return info references as a first phase of {@code receive-pack}.
+     *
+     * @since 1.0
+     */
+    static final class InforRefSlice extends Slice.Wrap {
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/git/ReceivePackSlice.java:48]: seems like there is a typo in the slice name

File: src/main/java/com/artipie/git/UploadPackSlice.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    UploadPackSlice() {
+        super(new SliceSimple(new RsWithStatus(RsStatus.NOT_IMPLEMENTED)));
+    }
+
+    /**
+     * References info phase for upload pack.
+     *
+     * @since 1.0
+     */
+    static final class InforRefSlice extends Slice.Wrap {
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/git/UploadPackSlice.java:49]: same here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@g4s8 thanks, several comments, take a look please

**Comment by g4s8** [comment]
@olenagerasimova thanks, all fixed

**Review by olenagerasimova — APPROVED** [review]
@g4s8 thanks, sorry for the delay

**Pull request merged**